### PR TITLE
add Unlock v10 docs

### PR DESCRIPTION
**Pull request by clemsos**

### CODE REVIEW

File: developers/smart-contracts/lock-api/README.md [hunk 1]
```diff
@@ -28,6 +28,8 @@ function initialize(
 ) external;
 ```
 
+NB: To create non-expiring memberships, just set the `_expirationDuration` to 0.
```
Comment (clemsos) [inline developers/smart-contracts/lock-api/README.md:31]: This is the only mention of non-expiring memberships I have added. Not sure if it should be expanded in the docs - and if yes where...

File: developers/smart-contracts/README.md [hunk 2]
```diff
@@ -46,7 +48,7 @@ We're making use of upgradeable contracts via [openzeppelin](https://docs.openze
 
 Good question. Until version 10, **Deployed locks are immutable**. That is, while they can be re-configured, disabled or destroyed _by their owner_, they will otherwise remain unchanged on the ethereum network for as long as it persists. Nobody else can modify a lock you deployed but you, unless you choose to allow this. After an upgrade of Unlock, all new locks deployed moving forward will be of the new version, and may support new features and/or improved usability.
 
-Starting with version 10, we are introducing upgradable locks. These locks are still deployed and owned by their creator, but they can later be upgraded to support new features introduced by the protocol, by their lock mnanagers only. The upgrades are optional and can only be triggered by the lock managers themselves.
+Starting with version 10, we are introducing upgradable locks. These locks are still deployed and owned by their creator, but they can later be upgraded to support new features introduced by the protocol, by their lock managers only. The upgrades are optional and can only be triggered by the lock managers themselves.
```
Comment (julien51) [inline developers/smart-contracts/README.md:51]:
```suggestion
Starting with version 10, we are introducing upgradable locks. These locks are still deployed and owned by their creator (who is the sole lock manager initially), but they can later be upgraded to support new features introduced by the protocol, by their lock managers only. The upgrades are optional and can only be triggered by the lock managers themselves.
```

File: developers/smart-contracts/lock-api/README.md (hunk 1 above)
Comment (julien51) [reply developers/smart-contracts/lock-api/README.md:31]: I think that's good for now, but we need to expose this on the front-end!

File: developers/smart-contracts/lock-api/README.md [hunk 3]
```diff
@@ -36,6 +38,7 @@ This function can be invoked to purchase a key (NFT membership).
 To purchase a membership, the caller should pass the number of tokens to be transfered (an ERC20 approval should have been initiated earlier), and this amount can be larger than the actual `keyPrice`.
 It is possible to purchase a membership on behalf of another user by setting a different recipient address.
 The referrer address is the address that will receive some UDT governance tokens as part of this transaction (if applicable).
+A key manager can also be set directly during the purchase, allowing a different account to manage the key while the recipient still benefit from a full membership.
```
Comment (julien51) [inline developers/smart-contracts/lock-api/README.md:41]: Maybe a link to the "Access Control" page could help here? (this is where we explain the roles)

**Review by julien51 — COMMENTED** [review]
LGTM but let's wait for @ccarfi and/or @desertrat10 to check the grammar 🇫🇷

**Review by ccarfi — APPROVED** [review]
thanks, clement! a few small changes, but nothing major. REALLY nicely done.

**Pull request merged**